build.sbt

```diff
@@ -9,7 +9,10 @@ val circeVersion = "0.14.5"
 val Log4jVersion = "2.20.0"
 
 libraryDependencies ++= Seq(
-  "com.amazonaws" % "amazon-kinesis-client" % "1.15.2" exclude("com.google.protobuf", "protobuf-java"),
+  "com.amazonaws" % "amazon-kinesis-client" % "1.15.2" exclude (
+    "com.google.protobuf",
+    "protobuf-java"
+  ),
```

Comment:
Annoying that these formatting changes were required! But I see why:

- the checks on [the PR that added these exclusions](https://github.com/guardian/fastly-cache-purger/pull/87) ran without the formatting test [and passed](https://github.com/guardian/fastly-cache-purger/actions/runs/11476359103/job/31936195710)
  - this is because it had branched off main before the check was introduced and wasn't up to date
  - to fix this I think we should change the CI trigger to `pull_request` and require PRs on this repo to be up to date with the latest main (more info in https://github.com/guardian/fastly-cache-purger/pull/90)
- once the PR was merged, [the build failed on main](https://github.com/guardian/fastly-cache-purger/actions/runs/11478015225/job/31941323876) and we didn't notice

This does mean though that the version of fastly-cache-purger in PROD doesn't include the changes from that old PR, which means merging this PR would produce a successful build and deploy a new version of the app that includes the changes from both PRs! So I'd like to first merge [this PR that fixes the formatting](https://github.com/guardian/fastly-cache-purger/pull/90), and make sure everything's working, then we can merge this PR.